### ISSUE TITLE
cli error due to cloudUploadOptions

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1977]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.13.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.13.0...4.13.1) (2021-08-03)

- build(deps-dev): bump @typescript-eslint/parser from 4.28.5 to 4.29.0 [`#1975`](https://github.com/open-wa/wa-automate-nodejs/pull/1975)
- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.28.5 to 4.29.0 [`#1976`](https://github.com/open-wa/wa-automate-nodejs/pull/1976)
- Release 4.13.1 [`1cdd671`](https://github.com/open-wa/wa-automate-nodejs/commit/1cdd67150df73c592d66b79b030f69ec76c6b82f)
- 🐛 fix: #1977 [`fc0dd5e`](https://github.com/open-wa/wa-automate-nodejs/commit/fc0dd5e09a77f5cfe955b4f9ed0757fc2743c046)
